tests/unit/test.program.js

```diff
@@ -60,7 +60,8 @@ describe('program.Program', () => {
       .command('cmd', 'some command', () => {
         throw new Error('this is an error from a command handler');
       });
-    return run(program, {systemProcess: fakeProcess, shouldExitProgram: true})
+    return execProgram(program, {systemProcess: fakeProcess,
+                                shouldExitProgram: true})
```

Comment:
The indent needs to align to somewhere after the brace of the opening statement. It needs to give a visual cue to the reader about which block is still open. Example:
````
return execProgram(program, {systemProcess: fakeProcess,
                             shouldExitProgram: true})
````
This would also work to give visual indication of which block is still open:
````
return execProgram(program, {
  systemProcess: fakeProcess,
  shouldExitProgram: true,
})
````
or this:
````
return execProgram(program, 
                   {systemProcess: fakeProcess, shouldExitProgram: true})
````